### PR TITLE
Rubocop Process Completed

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: spec/acceptance/version.rb [hunk 1]
```diff
@@ -32,20 +32,20 @@
   $vhost            = '/etc/apache2/sites-available/15-default.conf'
   $vhost_dir        = '/etc/apache2/sites-enabled'
   $run_dir          = '/var/run/apache2'
-  $doc_root          = '/var/www'
+  $doc_root = '/var/www'
```
Comment (pmcmaw) [inline spec/acceptance/version.rb]: Can you add in more spacing here to match the other equals signs above and below.

File: spec/acceptance/version.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@
   $vhost            = '/usr/local/etc/apache24/Vhosts/15-default.conf'
   $vhost_dir        = '/usr/local/etc/apache24/Vhosts'
   $run_dir          = '/var/run/apache24'
-  $doc_root          = '/var/www'
+  $doc_root = '/var/www'
```
Comment (pmcmaw) [inline spec/acceptance/version.rb]: Same as above.

File: spec/acceptance/version.rb [hunk 3]
```diff
@@ -68,7 +68,7 @@
   $vhost            = '/etc/apache2/vhosts.d/15-default.conf'
   $vhost_dir        = '/etc/apache2/vhosts.d'
   $run_dir          = '/var/run/apache2'
-  $doc_root          = '/var/www'
+  $doc_root = '/var/www'
```
Comment (pmcmaw) [inline spec/acceptance/version.rb]: Same as above.

**Comment by pmcmaw** [comment]
There is currently a failure on the adhoc pipeline.

File: tasks/init.rb [hunk 4]
```diff
@@ -5,19 +5,19 @@
 
 def service(action, service_name)
   if service_name.nil?
-    cmd_string = "facter -p osfamily"
-    stdout, stderr, status = Open3.capture3(cmd_string)
+    cmd_string = 'facter -p osfamily'
+    stdout, _stderr, _status = Open3.capture3(cmd_string)
```
Comment (pmcmaw) [inline tasks/init.rb]: This may need rejigged following that chat in standup this morning.

File: tasks/init.rb [hunk 5]
```diff
@@ -5,19 +5,19 @@
...
+                     'httpd'
+                   elsif osfamily == 'FreeBSD'
+                     'apache24'
+                   else
+                     'apache2'
+                   end
   end
   cmd_string = "service #{service_name} #{action}"
-  stdout, stderr, status = Open3.capture3(cmd_string)
+  _stdout, stderr, status = Open3.capture3(cmd_string)
```
Comment (pmcmaw) [inline tasks/init.rb]: Same as above.

File: spec/acceptance/mod_passenger_spec.rb [hunk 6]
```diff
@@ -80,164 +67,164 @@
...
           end
         when 'Debian'
           case fact('lsbdistcodename')
-          when 'wheezy'
-            it { is_expected.to contain "PassengerRuby \"#{passenger_ruby}\"" }
-            it { is_expected.not_to contain "/PassengerDefaultRuby/" }
           when 'jessie'
-            it { is_expected.to contain "PassengerDefaultRuby \"#{passenger_ruby}\"" }
-            it { is_expected.not_to contain "/PassengerRuby/" }
+            it { is_expected.to contain "PassengerDefaultRuby \"#{passenger_default_ruby}\"" }
```
[the remark this answers is not in the document]
Comment (david22swan) [reply spec/acceptance/mod_passenger_spec.rb]: It's the same value either way. I just changed the variable name to match what was in the text and removed the unused variables in each case.

[the remark this answers is not in the document]
Comment (pmcmaw) [reply spec/acceptance/mod_passenger_spec.rb]: Cool good shout! 👍

File: tasks/init.rb (hunk 4 above)
Comment (pmcmaw) [reply tasks/init.rb]: Never mind Hunter is working on this.

File: tasks/init.rb (hunk 5 above)
Comment (pmcmaw) [reply tasks/init.rb]: Never mind Hunter is working on this.

**Comment by david22swan** [comment]
Adhoc testing completed and passed: https://jenkins-master-prod-1.delivery.puppetlabs.net/view/modules/view/linux/view/_adhoc/view/apache/job/forge-module_puppetlabs-apache_intn-sys_smoke-adhoc/

**Pull request merged**